### ISSUE TITLE
Unable to resolve module AccessibilityInfo (RN v0.69.12)

### ISSUE THREAD

**Issue by nicosoutn2014** [juandl/react-native-credit-card-input-plus#16]
error: Error: Unable to resolve module AccessibilityInfo from /Users/.../node_modules/react-native-credit-card-input-plus/node_modules/react-native/Libraries/react-native/react-native-implementation.js: AccessibilityInfo could not be found within the project or in these directories:
  node_modules/react-native-credit-card-input-plus/node_modules

```
  17 | const ReactNative = {
  18 |   // Components
> 19 |   get AccessibilityInfo() { return require('AccessibilityInfo'); },
     |                                             ^
  20 |   get ActivityIndicator() { return require('ActivityIndicator'); },
  21 |   get ART() { return require('ReactNativeART'); },
  22 |   get Button() { return require('Button'); },
```

**Comment by jainminals** [comment]
error: Error: Unable to resolve module AccessibilityInfo from /Users/mac/Minal/FeelMotiRN/node_modules/react-native-credit-card-input-plus/node_modules/react-native/Libraries/react-native/react-native-implementation.js: AccessibilityInfo could not be found within the project or in these directories:
  node_modules/react-native-credit-card-input-plus/node_modules
  node_modules
  17 | const ReactNative = {
  18 |   // Components
> 19 |   get AccessibilityInfo() { return require('AccessibilityInfo'); },
     |                                             ^
  20 |   get ActivityIndicator() { return require('ActivityIndicator'); },
  21 |   get ART() { return require('ReactNativeART'); },
  22 |   get Button() { return require('Button'); },



getting same issue in react native "^0.68.7",

**Comment by jainminals** [comment]
"react-native": "^0.68.3" try this version of react native .

**Comment by DeyanDanailov** [comment]
The exact same error is reproduced also in RN 0.73.4 when <CreditCard component is rendered. @jainminals Please give some ideas for solving.
